neuland/executables/neulandAna.cxx

```diff
@@ -0,0 +1,129 @@
+#include "DigitizingChannelMock.h"
+#include "DigitizingPaddleMock.h"
+#include "DigitizingPaddleNeuland.h"
+#include "DigitizingTacQuila.h"
+#include "DigitizingTamex.h"
+#include "FairFileSource.h"
+#include "FairParRootFileIo.h"
+#include "FairRootFileSink.h"
+#include "FairRunAna.h"
+#include "FairRuntimeDb.h"
+#include "R3BNeulandDigitizer.h"
+#include "R3BNeulandHitMon.h"
+#include "TRandom3.h"
+#include "TStopwatch.h"
+#include <boost/program_options.hpp>
+
+using std::cout;
+using std::endl;
+
+using NeulandPaddle = Digitizing::Neuland::NeulandPaddle;
+using MockPaddle = Digitizing::Neuland::MockPaddle;
+using TamexChannel = Digitizing::Neuland::Tamex::Channel;
+using TacquilaChannel = Digitizing::Neuland::TacQuila::Channel;
+using MockChannel = Digitizing::Neuland::MockChannel;
+using Digitizing::UseChannel;
+using Digitizing::UsePaddle;
+
+const auto neulandEngines = std::map<std::pair<const std::string, const std::string>,
+                                     std::function<std::unique_ptr<Digitizing::DigitizingEngineInterface>()>>{
+    { { "neuland", "tamex" },
+      []() { return Digitizing::CreateEngine(UsePaddle<NeulandPaddle>(), UseChannel<TamexChannel>()); } },
+    { { "neuland", "tacquila" },
+      []() { return Digitizing::CreateEngine(UsePaddle<NeulandPaddle>(), UseChannel<TacquilaChannel>()); } },
+    { { "mock", "tamex" },
+      []() { return Digitizing::CreateEngine(UsePaddle<MockPaddle>(), UseChannel<TamexChannel>()); } },
+    { { "neuland", "mock" },
+      []() { return Digitizing::CreateEngine(UsePaddle<NeulandPaddle>(), UseChannel<MockChannel>()); } },
+    { { "mock", "mock" },
+      []() { return Digitizing::CreateEngine(UsePaddle<MockPaddle>(), UseChannel<MockChannel>()); } }
+};
+
+namespace po = boost::program_options;
+
+int main(int argc, const char** argv)
+{
+    TStopwatch timer;
+    timer.Start();
+
+    auto channelName = std::string{ "tacquila" };
+    auto paddleName = std::string{ "neuland" };
+    auto paraFileName = std::string{ "para.root" };
+    auto simuFileName = std::string{ "simu.root" };
+    auto digiFileName = std::string{ "digi.root" };
+    auto logLevel = std::string{ "error" };
+    auto eventNum = 0;
```

Comment:
what is eventNum? 
int, double, float?

Reply:
```cpp
auto var1 = 3;  // var1 is int
auto var2 = 3.; //var2 is double
auto var3 = 3.f; // var3 is float
auto var4 = 3L; // var4 is long
```
Please check the literal suffix in C++. It's very easy to understand if you know it.